### PR TITLE
Add PlayStation Plus 

### PR DESCRIPTION
**Pull request by domb84**
Fixes #(issue)
Added PlayStation Plus client installer

## Type of change
- [x] New installer
- [ ] Manifest fix
- [ ] Other

# Whas This Tested Using a [Local Repository](https://maintainers.usebottles.com/Testing)?
- [x] Yes
- [ ] No

### CODE REVIEW

**Comment by mirkobrombin** [comment]
Hi, thanks for your contribution.
Can you add the review for the new software and edit the eaclient one? https://github.com/bottlesdevs/programs/tree/main/Reviews

Remember that the Grade is the result of the lower grade in the review https://github.com/bottlesdevs/programs/blob/main/GUIDELINES.md#grades

**Comment by domb84** [comment]
@mirkobrombin done!

**Comment by domb84** [comment]
Removed changes to eaclient, purely includes PlaystionPlus client

File: Games/eaclient.yml
[the remark this answers is not in the document]
Comment (domb84) [reply Games/eaclient.yml:1]: Have removed

**Comment by orowith2os** [comment]
Lgtm, I'll leave Mirko to merge it

File: Reviews/eaclient.md [hunk 1]
```diff
@@ -32,4 +32,4 @@ Grade: Bronze
 Additional notes: n/a
 
 **Additional notes**  
-Need to be tested to more games but the experience is very smooth.
+Need to be tested to more games but the experience is very smooth. Removed seemingly unnecessary dependencies.
```
[the remark this answers is not in the document]
Comment (domb84) [reply Reviews/eaclient.md]: Also removed

**Pull request merged**